typescript/infra/config/environments/mainnet3/chains.ts

```diff
@@ -65,14 +65,17 @@ export const ethereumChainNames = Object.keys(
   ethereumMainnetConfigs,
 ) as MainnetChains[];
 
+// Remove mantapacific, as it's not considered a "blessed"
+// chain. It's not included in the scraper domains table,
+// and we don't relay to mantapacific on the Hyperlane or RC contexts.
+const hyperlaneContextRelayChains = ethereumChainNames.filter(
```

Comment:
should isRelayChain be a property in chainMetadata which you use to filter by? We might have more of these non-blessed chains in the future and it will quickly become cumbersome to maintain?

Reply:
Good point, I agree that this distinction is super confusing. I don't putting it in the SDK is the right move though as this is is specific to our infrastructure setup. I think the best solution here is to just put some love into supporting HaaS more appropriately in infra, atm it's a bit of a mess

Reply:
fair point, pointing it out as I have ran into similar issues bc our adhoc inclusion-exclusion of different chains in different parts of the codebase